apps/admin_api/priv/spec.yaml

```diff
@@ -204,6 +206,41 @@ paths:
           $ref: "#/components/responses/TokenResponse"
         '500':
           $ref: "#/components/responses/InternalServerError"
+  # Endpoint to create a new token
+  /token.stats:
+    post:
+      tags:
+        - Token
+      summary: Get the statistics for a token
+      operationId: token_stats
+      security:
+        - UserAuth: []
+      requestBody:
+        $ref: '#/components/requestBodies/TokenStatsBody'
+      responses:
+        '200':
+          $ref: "#/components/responses/TokenStatsResponse"
+        '500':
+          $ref: "#/components/responses/InternalServerError"
+
+  #####################################
+  #           MINTS ENDPOINTS         #
+  #####################################
+  /mint.all:
+    post:
+      tags:
+        - Mint
```

Comment:
If the endpoint is `/token.get_mints`, I'm not sure if we should tag with `Mint`. Maybe `Token`?

Or maybe tag with both `Token` and `Mint`?